### ISSUE TITLE
Fails with NPM global install

### ISSUE THREAD

**Issue by wbreza** [Microsoft/PowerBI-Cli#3]
When attempting to install globally `npm install -g powerbi-cli` you receive the following error:

```
> typings install && tsc
'typings' is not recognized as an internal or external command
```

**Comment by wbreza** [comment]
Workaround - Install typescript and typings globally first:
```
npm install -g typings typescript
```

**Comment by mattmazzola** [comment]
I think we could add typings and typescript as peer dependencies to warn people these are required although I'm not sure if the peer dependency lookup will know to check other global packages.

**Comment by wbreza** [comment]
Addressed in PR #4
